### ISSUE TITLE
Allow `tslint:disable:*`

### ISSUE THREAD

**Issue by leonard-thieu** [Microsoft/dtslint#120]
`@types/jquery` has [several test files](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/dc178ad6fe32416622ec37de703bb72279ab4bb4/types/jquery/README.md#project-structure) that contains code copied from jQuery documentation. They're useful in helping test `@types/jquery` but are written in JavaScript and are aligned to jQuery's style guidelines.

It'd be nice to be able to disable rules for these files without either disabling them for the entire package (which can negatively impact the quality of the declarations) or disabling them per line (which is a lot of work).

**Comment by andy-ms** [comment]
The problem with disabling *all* lint rules is that mosts of the tests dtslint does are written as lint rules. If you disabled all lint rules you would basically be testing nothing, while giving the impression that the types are well-tested. I think it's better to add individual rule disables to `tslint.json`.

**Comment by leonard-thieu** [comment]
Actually, I'm interested in disabling specific rules for specific files instead of all rules.

i.e.
```typescript
/* tslint:disable:rule1 rule2 rule3... */
```
